tests/test_audit_logs.py

```
@@ -0,0 +1,78 @@
+        self.assertEqual(len(response[0].authRequests), 0)
+
+    def test_list_auth_requests_pagination_token_is_client_validated_string(self):
+        """page_token accepts a string cursor; a malformed one is rejected by the server, not swallowed."""
+        # grpc_exec re-raises a malformed-cursor rejection (INVALID_ARGUMENT) as
+        # ScalekitServerException rather than returning an error-code response tuple — see
+        # GUIDELINES.md: "grpc_exec swallows grpc.RpcError — catch ScalekitServerException
+        # instead." Either an explicit rejection or a clean empty page is acceptable here —
+        # what's not acceptable is the SDK silently ignoring the parameter.
+        try:
+            response = self.scalekit_client.audit_logs.list_auth_requests(
+                page_token="not-a-real-cursor"
```

Ruff (0.15.21)

[error] Possible hardcoded password assigned to argument: "page_token"

(S106)